CommandAPI/commandapi-core/src/main/java/dev/jorel/commandapi/CommandAPICommand.java

```diff
@@ -392,26 +432,29 @@ public CustomCommandExecutor getExecutor() {
 		return executor;
 	}
 
-	/**
-	 * Sets the executors for this command
-	 * @param executor the executors for this command
-	 */
-	public void setExecutor(CustomCommandExecutor executor) {
-		this.executor = executor;
-	}
+	//================================================================================
+	//endregion
+	//region misc.
+	//================================================================================
 
 	/**
 	 * Returns whether this command is an automatically converted command
 	 * @return whether this command is an automatically converted command
 	 */
+
 	public boolean isConverted() {
 		return isConverted;
 	}
 
+	/**
+	 * @param isConverted
+	 * @todo Do we mark this as depricated? There are no javadocs for this so I'd assume its
+	 * for internal usage, but then shouldn't you mark as protected? I'm gonna leave this alone for now
+	 */
 	void setConverted(boolean isConverted) {
```

Comment:
This is marked with default access (which is stricter than public, hence doesn't have to be documented.